crates/wash-lib/src/cli/inspect.rs

```diff
@@ -85,32 +92,51 @@ pub async fn handle_command(
         .await?;
     }
 
-    if is_wasm(&buf) {
-        // Inspect an actor module
-        let module_name = command.target.clone();
-        let jwt_only = command.jwt_only;
-        let caps = get_caps(command.clone(), &buf).await?;
-        let token =
-            caps.ok_or_else(|| anyhow!("No capabilities discovered in : {}", module_name))?;
-        if jwt_only {
-            let out = CommandOutput::from_key_and_text("token", token.jwt);
-            Ok(out)
-        } else {
-            let validation = wascap::jwt::validate_token::<Actor>(&token.jwt)?;
-            let out = render_actor_claims(token.claims, validation);
-            Ok(out)
+    let output = match wasmparser::Parser::new(0).parse_all(&buf).next() {
+        // Inspect the WIT of a Wasm component
+        Some(Ok(wasmparser::Payload::Version {
+            encoding: wasmparser::Encoding::Component,
+            ..
+        })) if command.wit => {
+            let witty = wit_component::decode(&buf).expect("Failed to decode WIT");
+            let resolve = witty.resolve();
+            let main = witty.package();
+            let mut printer = wit_component::WitPrinter::default();
+            CommandOutput::from_key_and_text(
+                "wit",
+                printer
+                    .print(resolve, main)
+                    .expect("should be able to print WIT world from a component"),
```

Comment:
Should we panic here, or just `?` to return the error with this message as a `.context`?